docs/openid.md

```diff
@@ -89,7 +90,49 @@ oidc_config:
        localpart_template: "{{ user.name }}"
        display_name_template: "{{ user.name|capitalize }}"
 ```
+### [Keycloak][keycloak-idp]
 
+[Keycloak][keycloak-idp] is an opensource IdP maintained by Red Hat. 
+
+Follow the [Getting Started Guide](https://www.keycloak.org/getting-started/) to install Keycloak and set up a realm.
```

Comment:
This URL 404s for me?

Reply:
Strange, I thought I put the correct url. https://www.keycloak.org/docs/latest/getting_started/

Reply:
Done

Reply:
Wait, let me change the url again

Reply:
Yea, the original url was correct but I cannot link it from markdown. I find it strage.

Reply:
Done, I put an underscore by accident.